### DISCUSSION TITLE
Trailing whitespace visibility

### DISCUSSION THREAD

**Discussion by dbatten5** [arcticicestudio/nord#218] [Help & Support]
Is there a way to make trailing whitespace a bit more visible? It's quite difficult to spot it at a glance:

![image](https://user-images.githubusercontent.com/17271241/100518040-01793a00-3187-11eb-8850-58abd5c74c95.png)

**Comment by dbatten5** [comment]
ok found a solution to this:
```
augroup nord_theme_overrides
  autocmd!
  autocmd ColorScheme nord highlight ExtraWhitespace guifg=#88C0D0
  autocmd ColorScheme nord match ExtraWhiteSpace /\S\(\s\+\)$/
augroup END
```
happy to hear if there is a better way though

**Comment by arcticicestudio** [comment]
Hi @dbatten5 :wave:

Using `autocmd` groups is the recommended way to [customize Nord Vim][1]. Defining a custom highlighting group like you did is one way to match only specific tokens. To change the color for all non-printable characters ([`listchars`][2]), you can customize the `SpecialKey` group:

```vim
augroup nord-theme-overrides
  autocmd!
  " Use 'nord8' as foreground color for non-printable characters.
  autocmd ColorScheme nord highlight SpecialKey ctermfg=6 guifg=#88C0D0
augroup END
```

[1]: https://www.nordtheme.com/docs/ports/vim/customization
[2]: http://vimdoc.sourceforge.net/htmldoc/options.html#'listchars'

**Comment by svengreb** [comment]
@dbatten5 Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.
